cin_validator/rules/cin2022_23/rule_8869.py

```diff
@@ -0,0 +1,101 @@
+from typing import Mapping
```

Comment:
I notice that 8869 was mistakenly included here so I'll merge in main to prevent any conflicts before merging this in.